### ISSUE TITLE
Required ARIA child role not present for Multiselect dropdown while Fastpass Accessibility is testing.

### ISSUE THREAD

**Issue by Renuka8796** [microsoft/fluentui#18877]
**Environment Information:**
Package version(s): "@fluentui/react": "^8.22.3",
Browser and OS versions: chrome and Windows 10
**Describe the issue:**
Required ARIA child role not present for Multiselect dropdown while Fastpass Accessibility is testing.

Please provide a reproduction of the issue in a codepen: https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown

**Actual behavior:**
Required ARIA child role not present for Multiselect dropdown while Fastpass Accessibility is testing.
![image](https://user-images.githubusercontent.com/51446798/125065772-dc12be00-e0cf-11eb-99b8-03e994887ae6.png)

**Expected:**
Certain ARIA roles must contain particular children.

**Comment by ling1726** [comment]
This seems to violate WCAG 1.3.1. The current dropdown HTML output is a listbox wrapped around groups, which seems to be causing an issue with axe. I've tested with NVDA and Narrator, and the positions (1 of 7 etc...) are narrated correctly but in virtual cursor mode that's where you end up getting the group role and the headers.

@smhigley do you think this will cause a legitimate narration issue ?

```html
<div role="listbox>
  <div role="group">
    <div class="header" />
    <button role="option" />
    <button role="option" />
  </div>
</div>
```

**Comment by smhigley** [comment]
This is a false positive from Accessibility Insights and axe-core. Groups are allowed children of listboxes, and really the only way to correctly mark up a listbox with, well, multiple groups of options 😉.

I did some testing, and `role="listbox"` + `role="group"` performs as well or better than the native `<select>` + `<optgroup>`, and practical SR issues are minor. I definitely wouldn't suggest changing our current semantic structure (though we should probably add `role="presentation"` to the header).

The external issue against axe-core is here: https://github.com/dequelabs/axe-core/issues/2523
